src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -503,9 +504,9 @@ public bool TryGetRestClients(string requestPath, [MaybeNullWhen(false)] out Has
 
         private string? GetDefaultNameFromConfiguration(OperationSet operationSet, ResourceTypeSegment resourceType)
         {
-            if (MgmtContext.MgmtConfiguration.RequestPathToResourceName.TryGetValue(operationSet.RequestPath, out var name))
+            if (Configuration.MgmtConfiguration.RequestPathToResourceName.TryGetValue(operationSet.RequestPath, out var name))
```

Comment:
Maybe we should move `MgmtConfiguration` into separate static type?

Reply:
I prefer if we don't do this since all the plumbing for loading and saving would need to be duplicated.  We just wanted to wrap all of the mgmt specific configurations into a container for organization.